hudi-common/src/main/java/org/apache/hudi/common/bloom/BloomFilter.java

```diff
@@ -54,4 +54,10 @@ public interface BloomFilter {
    * @return the bloom index type code.
    **/
   BloomFilterTypeCode getBloomFilterTypeCode();
+
+  /**
+   * Performs a logical OR operations with other BloomFilter.
+   * @param other
+   */
+  void or(BloomFilter other);
```

Comment:
unit tests.

Reply:
In TestHoodieFileMetadataMerger#testBloomFilter

Reply:
@zhangyue19921010 can we move them to `TestBloomFilter` or the direct UT for the class.

Reply:
Add `TestBloomFilter#testBloomORMethod` cover `SIMPLE` and `DYNAMIC_V0` BloomFilterTypeCode